Source/Urho3D/Graphics/CameraOperator.h

```diff
@@ -0,0 +1,105 @@
+//
+// Copyright (c) 2024-2024 the rbfx project.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+/// \file
+
+#pragma once
+
+#include <Urho3D/Graphics/Camera.h>
+
+namespace Urho3D
+{
+/// Helper component that tracks points in world space and updates camera position.
+class URHO3D_API CameraOperator : public Component
```

Comment:
I would look for a better name. It should be such that unfamiliar user gets a rough idea what class does by reading the name. I do not have good suggestions though... `CameraAutoFitter`? Kind of meh...

Reply:
Won't fix. We can rename it later when someone comes up with better name